Herebyfile.mjs

```diff
@@ -138,36 +150,46 @@ function isInstalled(tool) {
     return !!which.sync(tool, { nothrow: true });
 }
 
+const builtLocal = "./built/local";
+
 const libsDir = "./internal/bundled/libs";
 const libsRegexp = /(?:^|[\\/])internal[\\/]bundled[\\/]libs[\\/]/;
 
-async function generateLibs() {
-    await fs.promises.mkdir("./built/local", { recursive: true });
+/**
+ * @param {string} out
+ */
+async function generateLibs(out) {
+    await fs.promises.mkdir(out, { recursive: true });

```

Comment:
Why not just import `fs/promises` instead of using `fs.promises` over and over?

Reply:
Because we also need the sync versions.

Reply:
Ugh, I thought that it would re-export those.